Frameworks/UIKit/UIActivityIndicatorView.mm

```diff
@@ -247,11 +259,13 @@ - (void)stopAnimating {
     RunSynchronouslyOnMainThread(^{
         if (_isAnimating) {
             _isAnimating = NO;
-        }
 
-        _progressRing.IsActive(false);
-        if (_hidesWhenStopped) {
-            [self setHidden:YES];
+            if (_hidesWhenStopped) {
+                // Make our view transparent, rather than hidden, to ensure space is reserved in the layout.
+                [self setAlpha:0.0];
```

Comment:
...you'd just need to plumb _hidesWhenStopped through so the backing Xaml control could do the right thing - it would simplify this code, too.

---
In reply to: [113292895](https://github.com/Microsoft/WinObjC/pull/2608#discussion_r113292895) [](ancestors = 113292895,113288024,113080688)